microsoft-365/security/office-365-security/tuning-anti-phishing.md

```diff
@@ -46,17 +46,21 @@ Specifically, you should check the **X-Forefront-Antispam-Report** header field
 
 - On a monthly basis, run [Secure Score](../mtp/microsoft-secure-score.md) to assess your organization's security settings.
 
-- Periodically review the [Spoof intelligence report](learn-about-spoof-intelligence.md) and [configure spoof intelligence](set-up-anti-phishing-policies.md#spoof-settings) to **Quarantine** suspicious messages instead of delivering them to the user's Junk Email folder.
+- For messages that end up in quarantine by mistake, or for messages that are allowed through, it is recommended to search for those messages in [Threat Explorer (Real-time detections)](threat-explorer.md). You can search by sender, recipient, or message id. Once you locate the message, go to details by clicking on the subject. For a quarantined mail, look to see what the "detection technology" was so that you can use the appropriate method to override. For an allowed mail, look to see which policy allowed the message. 
```

Comment:
```suggestion
- For messages that end up in quarantine by mistake, or for messages that are allowed through, it is recommended to search for those messages in [Threat Explorer (Real-time detections)](threat-explorer.md). You can search by sender, recipient, or message ID. Once you locate the message, go to details by clicking on the subject. For a quarantined mail, look to see what the "detection technology" was so that you can use the appropriate method to override. For an allowed mail, look to see which policy allowed the message. 
```

Reply:
Please change "message id" to message ID".